src/pages/meta-tag.js

```diff
@@ -67,14 +71,17 @@ export default function MetaTag(props) {
               setInvalidUrl('')
             }}
           />
-          <button
+          <Button
```

Comment:
Using the Material UI button component for consistency.
Also adding `submit` to make the keyboard enter work properly